packages/cropmarks/init.lua

```diff
@@ -7,107 +7,91 @@ local outcounter = 1
 
 local function outputMarks ()
   local page = SILE.getFrame("page")
-  SILE.outputter:drawRule(page:left() - 10, page:top(), -10, 0.5)
-  SILE.outputter:drawRule(page:left(), page:top() - 10, 0.5, -10)
-  SILE.outputter:drawRule(page:right() + 10, page:top(), 10, 0.5)
-  SILE.outputter:drawRule(page:right(), page:top() - 10, 0.5, -10)
-  SILE.outputter:drawRule(page:left() - 10, page:bottom(), -10, 0.5)
-  SILE.outputter:drawRule(page:left(), page:bottom() + 10, 0.5, 10)
-  SILE.outputter:drawRule(page:right() + 10, page:bottom(), 10, 0.5)
-  SILE.outputter:drawRule(page:right(), page:bottom() + 10, 0.5, 10)
+
+  -- Ensure the crop marks stay outside the bleed area
+  local offset = math.max(10, SILE.documentState.bleed / 2)
+
+  SILE.outputter:drawRule(page:left() - offset, page:top(), -10, 0.5)
+  SILE.outputter:drawRule(page:left(), page:top() - offset, 0.5, -10)
+  SILE.outputter:drawRule(page:right() + offset, page:top(), 10, 0.5)
+  SILE.outputter:drawRule(page:right(), page:top() - offset, 0.5, -10)
+  SILE.outputter:drawRule(page:left() - offset, page:bottom(), -10, 0.5)
+  SILE.outputter:drawRule(page:left(), page:bottom() + offset, 0.5, 10)
+  SILE.outputter:drawRule(page:right() + offset, page:bottom(), 10, 0.5)
+  SILE.outputter:drawRule(page:right(), page:bottom() + offset, 0.5, 10)
 
   SILE.call("hbox", {}, function ()
     SILE.settings:temporarily(function ()
       SILE.call("noindent")
       SILE.call("font", { size="6pt" })
-      SILE.call("crop:header")
+      if SILE.Commands["crop:header"] then
+        -- If user redefined this command, still use it with a warning...
+        SU.deprecated(" crop:header", "cropmarks:header", "0.14.0", "0.16.0")
+        SILE.call("crop:header")
+      else
+        SILE.call("cropmarks:header")
+      end
     end)
   end)
   local hbox = SILE.typesetter.state.nodes[#SILE.typesetter.state.nodes]
   SILE.typesetter.state.nodes[#SILE.typesetter.state.nodes] = nil
 
-  SILE.typesetter.frame.state.cursorX = page:left() + 10
-  SILE.typesetter.frame.state.cursorY = page:top() - 13
+  SILE.typesetter.frame.state.cursorX = page:left() + offset
+  SILE.typesetter.frame.state.cursorY = page:top() - offset - 3
   outcounter = outcounter + 1
 
   if hbox then
     for i = 1, #(hbox.value) do hbox.value[i]:outputYourself(SILE.typesetter, { ratio = 1 }) end
   end
 end
 
-local function reconstrainFrameset (fs)
```

Comment:
Why remove this? I realize you stopped using it, but couldn't we repurpose it to shim existing frameset configurations that are not setup relative to the page frame? This would ease the transition to the split paper/page setup if people didn't have to at the same time redefine all their framesets.

Reply:
Perhaps indeed. I just removed it because it was a local function, not exposed by the package, and now no longer used internally, but perhaps it could be moved e.g. to utilities (`SU.reconstrainFramesetToPage()` ?)

Reply:
I've been thinking about just automatically constraining all frames to the page *on instantiation* rather than "0,0" as the assumed default. Would that break anything? Then define "absolute" vs. "relative" as anything linked to the page/paper/trim/bleed frames rather than just page.